src/CRUD.js

```diff
@@ -36,7 +36,7 @@ exports.CRUD = class CRUD {
     return this.xhr(url, null, "DELETE");
   }
 
-  async xhr(url, body, verb) {
+  async xhr(url, body, verb, responseType = "json") {
```

Comment:
This solves the particular problem, though it feels quite specific for a utility library. It would be great if this method would support[ the entire request config from axios](https://github.com/axios/axios#request-config).

The short-hand methods (create, read, update, delete) can remain as they are, to keep it simple. So if we'd need create a special request with a certain responseType for instance, we could just do this:

```
const requestConfig = { responseType: 'blob', timeout: 10000 };
await CRUD.xhr('http://here.now', {}, 'POST', requestConfig)
```